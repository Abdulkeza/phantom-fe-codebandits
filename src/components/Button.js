import React from 'react';

const Button = ({ name, styles }) => {
  return (
    <div>
      <button type="button" className={`${styles} rounded-xl py-3 px-5`}>
        {name}
      </button>
    </div>
  );
};

export default Button;
