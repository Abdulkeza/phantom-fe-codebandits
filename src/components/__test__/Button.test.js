import React from 'react';
import renderer from 'react-test-renderer';
import Button from '../Button.js';

describe('Button', () => {
  it('should render Button', () => {
    const elem = renderer.create(<Button />).toJSON();
    expect(elem).toMatchSnapshot();
  });
});
