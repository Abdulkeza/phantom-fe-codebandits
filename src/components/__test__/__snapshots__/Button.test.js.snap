// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button it should render Button 1`] = `
<div>
  <button
    className="undefined rounded-xl py-3 px-5"
    type="button"
  />
</div>
`;
